### ISSUE TITLE
Is it a bug?

### ISSUE THREAD

**Issue by yiqiyu** [jsvine/pdfplumber#67]
I met this when trying to open a pdf:

```
Traceback (most recent call last):
  File "D:/P4/y/y/pledge_extraction/main.py", line 49, in read_pdf2
    with pdfplumber.open(file) as pdf:
  File "C:\Users\yyq\AppData\Local\Programs\Python\Python36-32\lib\site-packages\pdfplumber\pdf.py", line 40, in open
    return cls(open(path, "rb"), **kwargs)
  File "C:\Users\yyq\AppData\Local\Programs\Python\Python36-32\lib\site-packages\pdfplumber\pdf.py", line 34, in __init__
    self.metadata[k] = decode_text(v)
  File "C:\Users\yyq\AppData\Local\Programs\Python\Python36-32\lib\site-packages\pdfplumber\utils.py", line 68, in decode_text
    ords = (ord(c) if type(c) == str else c for c in s)
TypeError: 'bool' object is not iterable
```

It seems that pdfplumber isn't correctly handling boolean values in documents' metadata

**Comment by jsvine** [comment]
Hi @yiqiyu, and thanks for flagging. That does look like it might be a bug. Do you have an example of a PDF that raises this error?

**Comment by yiqiyu** [comment]
[ST东电_2018年第一季度报告全文_2018-04-28_1204826502.pdf](https://github.com/jsvine/pdfplumber/files/2135459/ST._2018._2018-04-28_1204826502.pdf)

Sure, @jsvine . Please see the attachment.

**Comment by Yevgnen** [comment]
Same here.

**Comment by jsvine** [comment]
Thanks again for flagging this and providing the example PDF. And thanks to @Yevgnen for the fix. Merged and now available in `v0.5.9`.
